Fix building with gcc-4.7

https://bugs.gentoo.org/show_bug.cgi?id=421871

Patch written by Kacper Kowalik <xarthisius@gentoo.org>
--- a/Source/util.h
+++ b/Source/util.h
@@ -18,6 +18,7 @@
 #define _UTIL_H_
 
 #include <string> // for std::string
+#include <unistd.h>
 
 #include "boost/scoped_ptr.hpp" // for boost::scoped_ptr
 #include "ResourceEditor.h"
